components/dot_menu/__snapshots__/dot_menu.test.tsx.snap

```diff
@@ -74,6 +74,11 @@ exports[`components/dot_menu/DotMenu should match snapshot, canDelete 1`] = `
       teamId="team_id_1"
     >
       <MenuItemAction
+        icon={
+          <span
+            className="icon-emoticon-plus-outline MenuItem__compass-icon"
+          />
+        }
         onClick={[Function]}
         show={false}
         text="Add Reaction"
```

Comment:
Is there a shortcut key for "Add Reaction"?

Reply:
Matt suggest to have no shortcut for `Add Reaction`